test/verify/machineslib.py

```diff
@@ -663,10 +666,16 @@ def fill(self):
                     if self.permanent:
                         b.click("#vm-{0}-disks-adddisk-permanent".format(self.vm_name))
                 else:
-                    # Choose storage pool
-                    b.select_from_dropdown("#vm-{0}-disks-adddisk-existing-select-pool".format(self.vm_name), self.pool_name)
-                    # Select from the available volumes
-                    b.select_from_dropdown("#vm-{0}-disks-adddisk-existing-select-volume".format(self.vm_name), self.volume_name)
+                    if self.source_type == "file":
```

Comment:
This option won't exist in images where libvirt-dbus is not present. Please add a check for that. Ex;
```
if m.image in ['ubuntu-1804]:
   wait_not_present(...
```

Reply:
Fixed